site/amoro-theme/layouts/partials/home-users.html

```diff
@@ -0,0 +1,41 @@
+<!-- - Licensed to the Apache Software Foundation (ASF) under one or more-->
+<!-- - contributor license agreements.  See the NOTICE file distributed with-->
+<!-- - this work for additional information regarding copyright ownership.-->
+<!-- - The ASF licenses this file to You under the Apache License, Version 2.0-->
+<!-- - (the "License"); you may not use this file except in compliance with-->
+<!-- - the License.  You may obtain a copy of the License at-->
+<!-- - -->
+<!-- -   http://www.apache.org/licenses/LICENSE-2.0-->
+<!-- - -->
+<!-- - Unless required by applicable law or agreed to in writing, software-->
+<!-- - distributed under the License is distributed on an "AS IS" BASIS,-->
+<!-- - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.-->
+<!-- - See the License for the specific language governing permissions and-->
+<!-- - limitations under the License.-->
+<div id="home-users">
+  <div class="content">
+    <div class="title">Who is Using Amoro</div>
+    <div class="subtitle">Trusted by leading companies worldwide</div>
+    <div class="users-wall">
+      <div class="users-track">
+        {{range .Site.Params.users}}
+        <div class="user-item" title="{{ .name }}">
+          <img src="{{ .Site.BaseURL }}/img/users/{{ .logo }}" alt="{{ .name }}" />
+        </div>
+        {{end}}
+        <!-- Duplicate for infinite scroll effect -->
+        {{range .Site.Params.users}}
+        <div class="user-item" title="{{ .name }}">
+          <img src="{{ .Site.BaseURL }}/img/users/{{ .logo }}" alt="{{ .name }}" />
+        </div>
+        {{end}}
+      </div>
+    </div>
+    <div class="add-company">
+      <a href="https://github.com/apache/amoro/issues/1853" target="_blank" class="btn btn-add">
```

Comment:
Do we need to create another issue page(how-to page) to track this `Add Your Company`? Currently, will jump to #1853, but this content here we need to update manually.

The `how-to` page show the users how to add this content, so that they can file an pr directly.